server/src/test/java/org/opensearch/index/query/RangeQueryBuilderTests.java

```diff
@@ -185,7 +188,7 @@ protected void doAssertLuceneQuery(RangeQueryBuilder queryBuilder, Query query,
             } else if (expectedFieldName.equals(DATE_FIELD_NAME)) {
                 assertThat(query, instanceOf(IndexOrDocValuesQuery.class));
                 query = ((IndexOrDocValuesQuery) query).getIndexQuery();
-                assertThat(query, instanceOf(PointRangeQuery.class));
```

Comment:
should this unpack and assert ApproximateableQuery  also contains a PointRangeQuery?